cmd/gapir/cc/main.cpp

```diff
@@ -393,12 +396,16 @@ void android_main(struct android_app* app) {
     }
 
     if (serverIsDone) {
-      unlink(socket_file_path.c_str());
-      alive = false;
+      // Start termination of the app
+      ANativeActivity_finish(app->activity);
```

Comment:
This now gets executed on every iteration, as soon as serverIsDone is true. Setting it back to false seems a bit dirty, but would work.

Reply:
Good point! I added a "finishing" boolean to avoid calling ANativeActivity_finish() several time.